console/src/main/scripts/plugins/metrics/ts/directives/overview/statusOverviewDirective.ts

```diff
@@ -23,26 +23,17 @@ module HawkularMetrics {
     constructor(private $filter: ng.IFilterService) {}
 
     public parseDayFromTimestamp(timestamp): string {
-      const dateFilter = this.$filter('date');
-      const parsedDate = new Date(timestamp);
-      const today = new Date();
-      const yesterday = new Date();
-      yesterday.setDate(today.getDate() - 1);
-      if (StatusOverviewController.isSameDayMonthYear(parsedDate, yesterday)) {
-        return 'Yesterday';
-      } else if (StatusOverviewController.isSameDayMonthYear(parsedDate, today)) {
+      const parsedDate = moment(timestamp).startOf('day');
+      const today = moment().startOf('day');
+      if (today.diff(parsedDate) === 0) {
         return 'Today';
+      } else if (today.add(-1, 'days').startOf('day').diff(parsedDate) === 0) {
+        return 'Yesterday';
```

Comment:
Well the `calendar()` is used like: `moment().add(-1, 'days').calendar()` and the output is `Yesterday at 4:28 PM` I don't think we want PM/AM in there. But I can still change settings of calendar and use this instead returning string 'Yesterday' or 'Today'